### ISSUE TITLE
🚀 Feature: Override max file size limit on a per user basis

### ISSUE THREAD

**Issue by avesst** [stonith404/pingvin-share#333]
### 🔖 Feature description

Ability to override the global max file size limit on a per user basis.

### 🎤 Pitch

I would like to share my instance with family and friends have the global max file size set to 1GB while personally being able to upload larger files.

**Comment by stonith404** [comment]
Might #103 be related to this?

**Comment by maetthew** [comment]
Yes and no. That request is about he total uploaded size of a user, whereas this pertains to a single file size limit of a user. I guess they are related but I would consider them two different features.

**Comment by blackletum** [comment]
This is what I was hoping for as well, hopefully it gets implemented soon. I have a few files that are multiple GB large I'd like to share with people externally from my server at home.
